### PR TITLE
Create 0000-01-02-ZackConord.md

### PR DESCRIPTION
**Pull request by ZackConord**
Add intro file requested by the course.

### CODE REVIEW

**Review by robert-crandall — APPROVED** [review]
Welcome to an approved PR 😄

**Review by robert-crandall — APPROVED** [review]
Oh, I don't have write access. I can't approve this - sorry!

**Comment by ZackConord** [comment]
@robert-crandall Thanks for trying!

**Review by pwoosam — APPROVED** [review]

**Comment by pwoosam** [comment]
It looks like you'll have to invite one of us as a collaborator to give us write access to this repo

1. Open this repo's`Settings` tab (near the top of this page)
2. Click `Manage access` in the side-menu to the left
3. Invite a collaborator

File: _posts/0000-01-02-ZackConord.md
Comment (ZackConord) [inline _posts/0000-01-02-ZackConord.md:5]:
```suggestion
DevSecOps
```

**Review by ZackConord — COMMENTED** [review]
Perhaps "DevSecOps vs DevOps"

**Comment by ZackConord** [comment]
Test

**Comment by ZackConord** [comment]
> Perhaps "DevSecOps vs DevOps"

test

File: _posts/0000-01-02-ZackConord.md
Comment (ZackConord) [reply _posts/0000-01-02-ZackConord.md:5]: Should we update with "DevSecOps vs DevOps"

**Review by ZackConord — COMMENTED** [review]
I'd like to leave as is.  I plan to address DevOps on another slide deck.

**Review by ZackConord — COMMENTED** [review]
Testing again - I apologize to anyone receiving notifications on this test repo! 😄

File: _posts/0000-01-02-ZackConord.md
Comment (ZackConord) [reply _posts/0000-01-02-ZackConord.md:5]: Let's hold off for now.

**Review by ZackConord — COMMENTED** [review]
test

**Review by ZackConord — COMMENTED** [review]
Testing again 😃

**Review by ngonz — APPROVED** [review]
LGTM!

**Pull request merged**
